src/Forms/PrepareLetterPage.jsx

```diff
@@ -7,21 +7,39 @@ import PropTypes from "prop-types";
 import { useAmplitude } from "../context/amplitude";
 import { genererFoerstesideData, getVedleggsFelterSomSkalSendes } from "../util/forsteside";
 import { lastNedFilBase64 } from "../util/pdf";
-import { AppConfigContext } from "../configContext";
+import { useAppConfig } from "../configContext";
+import TEXTS from "../texts";
+import { useLanguages } from "../context/languages";
 
-const LeggTilVedleggSection = ({ index, vedleggSomSkalSendes }) => {
+const LeggTilVedleggSection = ({ index, vedleggSomSkalSendes, translate }) => {
   const skalSendeFlereVedlegg = vedleggSomSkalSendes.length > 1;
   return (
     <section
       className="wizard-page"
-      aria-label={`${index}. Legg ved ${skalSendeFlereVedlegg ? "disse vedleggene" : "dette vedlegget"}.`}
+      aria-label={`${index}. ${translate(
+        TEXTS.prepareLetterPage.attachmentSectionTitleAttachTo
+          .concat(" ")
+          .concat(
+            skalSendeFlereVedlegg
+              ? TEXTS.prepareLetterPage.attachmentSectionTitleTheseAttachments
+              : TEXTS.prepareLetterPage.attachmentSectionTitleThisAttachment
+          )
+      )}`}
     >
       <Systemtittel className="margin-bottom-default">
-        {index}. Legg ved {skalSendeFlereVedlegg ? "disse vedleggene" : "dette vedlegget"}
+        {`${index}. ${translate(
+          TEXTS.prepareLetterPage.attachmentSectionTitleAttachTo
```

Comment:
Repeats code from just above. Extract to a const before the return.

Reply:
Fixed